xml/templates/message_templates/event_registration_receipt_html.tpl

```diff
@@ -25,7 +25,7 @@
     {/if}
 
     <p>Your order number is #{$transaction_id}. {if !empty($line_items) && empty($is_refund)} Information about the workshops will be sent separately to each participant.{/if}
-  Here's a summary of your transaction placed on {$transaction_date|date_format:"%D %I:%M %p %Z"}:</p>
+  Here's a summary of your transaction placed on {$transaction_date|crmDate:"%D %I:%M %p %Z"}:</p>
```

Comment:
Very different.

Reply:
See above about %Z. Otherwise there's a minor difference now but unlikely to be a problem. Previously: "Eastern Daylight Time", now: "EDT". And am/pm vs AM/PM.

Reply:
@demeritcowboy ok so the lower case AM/PM might be a thing but the timezone to me looks like it as always meant to be abbreviation as per https://www.php.net/manual/en/function.strftime.php note it is capital Z

Reply:
Yeah ok I'm not too worried about this difference. It's more that date('T') will give the wrong zone half the time since it is always based on "now" (because of pesky Daylight Saving).